### DISCUSSION TITLE
whisper.load_audio(abs_file_path) and  model.transcribe(abs_file_path) throws error

### DISCUSSION THREAD

**Discussion by im-kapil** [openai/whisper#1474] [Q&A] [answered]
Hello there developers and community members, 

I am blockchain developer and just put my feet inside the world of AI. 

So first I am trying to create a AI based project for my self. 

So my first requirement is to create audio transcribe. Where I want to transcribe complete audio file which is in English language to text, to perform some operation on that text. 

For it I am supposed to use **Whisper Model**  But I get one error and the error was **[WinError 2] The system cannot find the file specified** 

I have my file inside the same directory where my python code is. 

**Here is my code with different approaches I tried** 

**Approach 1** 
```

import whisper
import os

# Get the absolute path of the current script file
script_dir = os.path.dirname(os.path.abspath(__file__))
# Specify the filename
filename = 'podcast.mp3'
# Construct the absolute path to the audio file
abs_file_path = os.path.join(script_dir, filename)
print("abs_file_path:::::", abs_file_path)
# Load the model
model = whisper.load_model("base")
# Load audio and pad/trim it to fit 30 seconds
audio = whisper.load_audio(abs_file_path)
```

**Approach 2**
```

import whisper
# Load the model
model = whisper.load_model("base")
# Load audio and pad/trim it to fit 30 seconds
# I also tried it this way whisper.load_audio('./podcast.mp3') and whisper.load_audio('/podcast.mp3')
audio = whisper.load_audio('podcast.mp3')
# audio = whisper.pad_or_trim(audio)
```

**Approach 3**

In this approach I tried with different code
```

import whisper
model = whisper.load_model("base")
result = model.transcribe("podcast.mp3")
print(result["text"])
```

I tried with different ways but I get the same error again and again **FileNotFoundError: [WinError 2] The system cannot find the file specified**

Here is my project hierarchy 

![image](https://github.com/openai/whisper/assets/89021712/ae9579f2-23ca-47c0-a66c-82e480d00bc3)

OS I am using: **Windows11**

Now looking for some help. 

Thanks

**Comment by glangford** [comment, marked as answer]
fyi in case this is the source of your problem
* https://github.com/openai/whisper/discussions/1468
* https://github.com/openai/whisper/discussions/109

**Reply by im-kapil** [reply]
Hello @glangford, 

This solution worked for me, I installed ffmpeg using Chocolatey and now ffmpeg successfully installed. 

Thanks
